### ISSUE TITLE
Adding DataService to Big Sample

### ISSUE THREAD

**Issue by speedpidi** [DEVDEERZ/DoctorFlox#76]
Is it possible to add a DataService to the big sample app. My idea was to show how communication to a external Unit oder a Database can be made. Maybe it is helpful to pick a dispatcher timer, let it run in the dataservice and change the percentage of the progress bar. I think that this can explain how communication with external data can be realized.

**Comment by devdeer-alex** [comment]
Good idea. Any suggestions for a public data service we could use? Should be one that delivers a significant amount of data.

**Comment by speedpidi** [comment]
Maybe this helps.
https://github.com/toddmotto/public-apis#development
I think there is a problem with using a public database, cause everything in the web is changing every day. Now you choose a database and tomorrow the sample app is broken, because the database is gone.

There are free Generators online were you can easily create data. I think it is a better idea to read the data from a XML File which the user can create in the Internet. Or something like this.

**Comment by devdeer-alex** [comment]
#78 is ongoing which should solve this.

**Comment by speedpidi** [comment]
DataService is broken cause adsbexchange closed public api. Models and everything is outdated.
